compiler/onecc-docker/README.md

```diff
@@ -0,0 +1,263 @@
+# onecc-docker
+
+`onecc-docker` supports `one-cmd`, which is currently only supported by ubuntu 18.04, in more OS.
```

Comment:
```suggestion
`onecc-docker` supports `one-cmds`, which is currently only supported on ubuntu 18.04, in more OS.
```

Reply:
Q) what is `in more OS.` ?

Reply:
> Q) what is in more OS. ?

More OSs can provide it.

Reply:
This means that `onecc-docker` supports running `onecc` in environments other than ubuntu:18.04.

Reply:
How about
- _onecc-docker_ can expand `one-cmds` in more platforms than just Ubuntu 18.04/20.04.
- or
- We can use `one-cmds` in more platorms than just Ubuntu 18.04/20.04 through _onecc-docker_.